R/standardize_subject_ids.R

```diff
@@ -152,23 +137,23 @@ check_subject_ids <- function(data,
 #' # perform the correction
 #' dat <- correct_subject_ids(
 #'   data             = dat,
-#'   id_column_name   = "study_id",
+#'   target_columns   = "study_id",
 #'   correction_table = correction_table
 #' )
-correct_subject_ids <- function(data, id_column_name, correction_table) {
+correct_subject_ids <- function(data, target_columns, correction_table) {
   checkmate::assert_data_frame(correction_table, any.missing = FALSE,
                                min.rows = 1L, ncols = 2L, null.ok = FALSE)
 
   stopifnot("Some ids in the correction table were not found in the input data"
```

Comment:
The following table will fail silently
```r
correction_table <- data.frame(
  random_name = c("P0005P2", "PB500P2", "PS004P2-1"),
  other_random_name   = c("PS001P2", "PB050P2", "PS004P2")
)
```

because `correction_table` requires colNameas `c("from", "to")`.

Consider checking the column names and adding a test to check for it.

Reply:
This has been addressed in commit `3d0aafb`.